packages/material-ui/src/layout.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Paper from 'material-ui/Paper';
+
+const Layout = ({ children }) => (
+  <Paper style={{ margin: 'auto', maxWidth: 400, marginTop: 50, padding: 20 }}>
```

Comment:
So this is the default layout right? but then, I don't see any way of overriding it in a comfortable way